cmd/import.go

```diff
@@ -22,44 +22,19 @@ var importCmd = &cobra.Command{
 	Use:   "import",
 	Short: "Import media",
 	Run: func(cmd *cobra.Command, args []string) {
-		input, err := cmd.Flags().GetString("input")
-		if err != nil {
-			cui.Error("Problem parsing input", err)
-		}
-		output, err := cmd.Flags().GetString("output")
-		if err != nil {
-			cui.Error("Problem parsing output", err)
-		}
-		camera, err := cmd.Flags().GetString("camera")
-		if err != nil {
-			cui.Error("Problem parsing camera", err)
-		}
-		projectName, err := cmd.Flags().GetString("name")
-		if err != nil {
-			cui.Error("Problem parsing name", err)
-		}
+                input := get_flag_string(cmd, "input")
+                output := get_flag_string(cmd, "output")
+                camera := get_flag_string(cmd, "camera")
+                projectName := get_flag_string(cmd, "name")
```

Comment:
What do you think about making `--name` mandatory to be passed via CLI each time? So videos don't get accidentally sent to an old project, for instance.

Reply:
If a user doesn't have "projects?" and uses camera/date or even the video file name?
maybe it should be combination with something?

Reply:
True, lets keep it the way it is.